geometry_msgs/PoseStamped[] palm_poses_in_obj
string tf_prefix
---
bool success
